src/Files.App/Views/Properties/GeneralPage.xaml

```diff
@@ -333,7 +333,7 @@
 							Grid.ColumnSpan="2"
 							Height="1"
 							Margin="-16,0"
-							Background="{ThemeResource DividerStrokeColorDefault}" />
+							Background="{ThemeResource DividerStrokeColorDefaultBrush}" />
```

Comment:
I'll have to double check but I seem to recall this causing issues when switching between light and dark mode.

Reply:
Is that so? I just looked over git blames and it seems that DividerStrokeColorDefault line has not updated since I changed to this and this seems to had been DividerStrokeColorDefault**Brush** before I change. The same for the other changes on this brush.